webhooks/README.md

```diff
@@ -1,30 +1,30 @@
-You can create up to five webhook URLs for the same event in the same Program. Fidel sends the data via HTTP POST request and will send test or live events depending on your dashboard's environment setting or if you are using test or live API keys.
+You can create up to five webhook URLs for the same event in the same Program. Fidel sends the data via HTTP POST in JSON format and will send test or live events depending on your dashboard's environment setting or if you are using test or live API keys.
 
 Fidel only accepts HTTPS URLs for webhooks endpoints. In order to create webhooks and receive event data, your server must be configured to support HTTPS with a valid certificate.
 
 <div class="info-box">
-    <small>Important note</small><br/>
+    <small>Return a 200 status code</small><br/>
 	To confirm receipt of a webhook event, your server endpoint should return a <code>200</code> HTTP status code. Any other response will be treated as a failure and we retry the request three times over the next hour with exponential back off.
 </div>
+---
 
-## Authentication
-
+## Signatures
 To confirm that received events are being sent from Fidel we recommend verifying webhook signatures. That can be done by using the `x-fidel-signature` and `x-fidel-timestamp` HTTP headers. This isn't required, but offers an additional layer of security.
 
----
-
-A unique secret key is generated for each webhook. The key is returned in the response's `secretKey` property if you are using the Webhooks API. You can also copy the key from the dashboard's webhooks page by clicking in the **Show Key** button next to your webhook endpoint. To verify a webhook request, generate a signature using the same key that Fidel uses and compare that to the value of the `x-fidel-signature` header.
+A unique secret key is generated for each webhook. The key is returned in the response's `secretKey` property if you are using the Webhooks API. You can also copy the key from the dashboard's webhooks page by clicking in the **Show Key** button next to your webhook endpoint. To verify a webhook request, generate a signature using the same key that Fidel uses and compare that to the value of the `x-fidel-signature` header:
 
-1. Create a string concatenating the body of the request, the webhook URL and the timestamp value from the `x-fidel-timestamp` header.
-2. Double hash the resulting string using the webhook key with HMAC-SHA256 and encode it in Base-64.
-3. Compare the signature you generated with the signature provided in the `x-fidel-signature` header.
+> **1.** Create a string concatenating the body of the request, the webhook URL and the timestamp value from the `x-fidel-timestamp` header.
+> **2.** Double hash the resulting string using the webhook key with HMAC-SHA256 and encode it in Base-64.
+> **3.** Compare the signature you generated with the signature provided in the `x-fidel-signature` header.
```

Comment:
let's keep an eye out for how this renders

Reply:
Yes :) This is an experiment. Let's see how it looks.
Do we still have an develop environment setup. Would be good if we could deploy first to dev to see the changes before merging to master.

Reply:
not now, we're moving to a different flow. shouldn't be a problem, I'd expect it to
> **1.** render as a callout
> **2.** like this

i'm confident enough, but here to fix if it breaks